app/Http/Controllers/Api/V1/CurrikiGo/LmsController.php

```diff
@@ -66,4 +69,37 @@ public function projects(Request $request)
             'projects' => ProjectPublicResource::collection($projects),
         ], 200);
     }
+
+    public function activities(Request $request)
+    {
+        // Check LMS settings for authorization
+/*
+        $settings = $this->lmsSettingRepository->fetchByLmsUrlAndLtiClientId($request->lms_url, $request->lti_client_id);
+        if(empty($settings))
+            return response(['errors' => ['Unauthorized']], 401);
+*/
```

Comment:
If you want to keep this as a reference then it's okay, otherwise, you could remove it.
Use opening/closing braces for if/else

Reply:
Keeping it since its going to be relevant in the next features discussed with mike